src/iosrv/drpc_client.c

```diff
@@ -51,6 +52,14 @@ notify_ready(void)
 	if (rc != 0)
 		goto out;
 	req.nctxs = DSS_CTX_NR_TOTAL;
+	/* Do not free, this string is managed by the dRPC listener */
+	req.drpclistenersock = drpc_listener_socket_path;
+	/*
+	 * TODO DAOS-2881: Use idx that is passed from daos_server on startup.
```

Comment:
Are you looking at adding a new command-line flag for this?

Reply:
Yeah, that is what I was thinking. I could add it now, or in a follow-on patch.

Reply:
Seems like related work and will make testing this easier if you add it now. I don't feel strongly about it though, if you see a good reason for splitting it out, that's fine too.